Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppMgmt/IISWebAppMgmtV1/Main.ps1

```diff
@@ -30,9 +30,10 @@ param (
     [string]$deployInParallel
```

Comment:
Change the data type(string to int) of iisVersion parameter in Add-SslCert function of AppCmdOnTargetMachines.ps1 file of V1 task.

Reply:
Done